docs/support-guide.md

```diff
@@ -0,0 +1,261 @@
+- Archive with Certification results
+- Archive with must-gather
+- [Installation Checklist (file `user-installation-checklist.md`)](./user-installation-checklist.md) with the partner's update to sign off post-instalation items
+
+### New Executions <a name="check-list-new-executions"></a>
+
+The following assets, certification assets, should be updated when certain conditions happen:
+
+- Certification Results
+- Must Gather
+- Install Documentation (when any item/flavor/configuration has been modified)
+
+
+The following conditions require new certification assets:
+
+- The version of the OpenShift Container Platform has been updated
+- Any Infrastructure component(s) (e.g.: server size, disk category, ELB type/size/config) or cluster dependencies (e.g.: external storage backend for image registry) have been modified
+
+
+## Review Environment <a name="setup"></a>
+
+### Setting up local environment <a name="setup-install"></a>
+
+- Download the [OPCT](./user.md#install)
+- Download the [`omg`](https://github.com/kxr/o-must-gather)
+
+```bash
+pip3 install o-must-gather --user
+```
+
+### Download dependencies <a name="setup-download-dependencies"></a>
+
+- Download the Baseline results from the <TBD>`quay.io/ocp-cert/baseline-results`
+
+> TODO: add the steps to pull and extract the results from the baseline repository
+
+- Download the suite test list for the version used by the partner
+
+```bash
+RELEASE_VERSION="4.11.4->CHANGE_ME"
+TESTS_IMG=$(oc adm release info ${RELEASE_VERSION} --image-for='tests')
+oc image extract ${TESTS_IMG} --file="/usr/bin/openshift-tests"
+chmod u+x ./openshift-tests
+./openshift-tests run --dry-run kubernetes/conformance > ./test-list_openshift-tests_kubernetes-conformance.txt
+./openshift-tests run --dry-run openshift/conformance > ./test-list_openshift-tests_openshift-validated.txt
+```
+
+### Download Partner Results <a name="setup-download-results"></a>
+
+- Download the Provider certification archive from the Support Case. Example file name: `retrieved-archive.tar.gz`
+- Download the Must-gather from the Support Case. Example file name: `must-gather.tar.gz`
+
+## Review guide: exploring the failed tests <a name="review-process"></a>
+
+The steps below use the subcommand `process` to apply filters on the failed tests and help to keep the initial focus of the investigation on the failures exclusively on the partner's results.
+
+The filters use only tests included in the respective suite, isolating from common failures identified on the Baseline results or Flakes from CI. To see more details about the filters, read the [dev documentation describing filters flow](./dev.md#dev-diagram-filters).
+
+Required to use this section:
+- OPCT CLI downloaded to the current directory
+- OpenShift e2e test suite exported to the current directory
+- Baseline results exported to the current directory
+- The Certification Result is in the current directory
+
```

Comment:
This does not render as a list when previewing in `mkdocs serve`

Reply:
Yeah, need a new line before the first bullet. I will fit it. Thanks